docs/sphinx/user_guide/feature/reduction.rst

```diff
@@ -300,9 +358,10 @@ any number of ``RAJA::expt::Reduce`` objects to the ``RAJA::forall`` method::
   std::cout << rm_loc.getLoc() ...
 
 Again, the lambda expression parameters are in the same order as
-the ``RAJA::expt::Reduce`` arguments to ``RAJA::forall``. Both the types and
-order of the parameters must match to get correct results and to compile
-successfully. Otherwise, a static assertion will be triggered::
+the ``RAJA::expt::Reduce`` arguments to ``RAJA::forall``. The ``ValOp`` underlying
+data types and operators, and order of the ``ValOp`` parameters must match 
+the corresponding ``RAJA::expt::Reduce`` types to get correct results and to
+compile successfully. Otherwise, a static assertion will be triggered::
 
   LAMBDA Not invocable w/ EXPECTED_ARGS.
```

Comment:
`EXPECTED_ARGS`?? Is that clear?

Reply:
That's searchable and clear for us internally. I can update this in the code to be a little more helpful.

Reply:
But it's in the user guide....

Reply:
Yes, I've updated this in both the code and user guide to be more helpful.